docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```bash
+npm install \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+</Tabs>
+
+
+
+### Instantiating an Ethers Signer
+The `ETHEREUM_PRIVATE_KEY` environment variable is required.
+```ts
+import { LIT_RPC } from "@lit-protocol/constants";
+import * as ethers from "ethers";
+
+const ethersSigner = new ethers.Wallet(
+  ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+```
+
+### Instantiating a `LitNodeClient`
+Here we are instantiating an instance of `LitNodeClient` and connecting it to the `datil-dev` Lit network. To simulate a Node.js environment, we use `LocalStorage` from the `node-localstorage` module to store our session keys and metadata.
+
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork } from "@lit-protocol/constants";
+import { LocalStorage } from "node-localstorage";
+
+let litNodeClient: LitNodeClient;
+
+litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilDev,
+      debug: false,
+      storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+      },
+    });
+await litNodeClient.connect();
+```
+
+### Generating Session Signatures
+The session signatures generated in this example provide unrestricted access to all resources. However, in real-world applications, it's more common and secure to limit access. Typically, you would use the specific hash of the encrypted access control condition resource key, granting access only to particular resources rather than everything. To this resource, we then give it the ability to decrypt an encryption access control condition. 
+
+```ts
+import {
+  LitAbility,
+  LitAccessControlConditionResource,
+  createSiweMessage,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+const sessionSignatures = await litNodeClient.getSessionSigs({
+  chain: "ethereum",
+  expiration: new Date(Date.now() + 1000 * 60 * 10 ).toISOString(), // 10 minutes
+  resourceAbilityRequests: [
+    {
+      resource: new LitAccessControlConditionResource("*"),
+      ability: LitAbility.AccessControlConditionDecryption,
+    },
+  ],
+  authNeededCallback: async ({
     uri,
     expiration,
     resourceAbilityRequests,
   }) => {
-    // Prepare the SIWE message for signing
-    const toSign = await createSiweMessageWithRecaps({
-      uri: uri,
-      expiration: expiration,
+    const toSign = await createSiweMessage({
+      uri,
+      expiration,
       resources: resourceAbilityRequests,
-      walletAddress: wallet.address,
+      walletAddress: await ethersSigner.getAddress(),
```

Comment:
```suggestion
      walletAddress: wallet.address,
```

This should work

Reply:
Wallet isn't defined in the code example, ethersSigner.address seems to be alright